doc/15-troubleshooting.md

```diff
@@ -317,6 +317,116 @@ not `assign where match("MySQL*", host.name) && host.vars.production`.
 For all test hosts `host.vars.production` will be false and terminate
 the `&&` rather than also evaluating `match("MySQL*", host.name)`.
 
+### Try reducing concurrency (threads amount)
```

Comment:
Something general I noticed multiple times while reading over this: "amount" isn't used with countable things: https://dictionary.cambridge.org/grammar/british-grammar/amount-of-number-of-or-quantity-of